src/containers/Tenant/Query/ExplainResult/ExplainResult.js

```diff
@@ -141,7 +141,7 @@ export function ExplainResult(props) {
         const content = (
             <div className={b('ast')}>
                 <MonacoEditor
-                    language={LANGUAGE_S_EXPRESSION_ID}
+                    language={LANGUAGE_ID}
```

Comment:
It doesn't work for ast

Before:
<img width="1179" alt="Screenshot 2024-07-25 at 14 09 50" src="https://github.com/user-attachments/assets/1ef86c99-4049-4954-b507-75c48a13c274">


Now:
<img width="1235" alt="Screenshot 2024-07-25 at 14 05 40" src="https://github.com/user-attachments/assets/766fe0b6-4420-4f6f-8206-178faf7817c2">